utils/debug.c

```diff
@@ -230,6 +230,10 @@ void __pr_color(char code, const char *fmt, ...)
 
 	va_start(ap, fmt);
 	vfprintf(outfp, fmt, ap);
+	if (logfp != stderr) {
+		va_start(ap, fmt);
+		vfprintf(logfp, fmt, ap);
+	}
```

Comment:
But this will add other (normal) messages to the logfile.  I think you'd better changing the segfault code to use `pr_warn()` instead.